functions/__sf_section_host.fish

```diff
@@ -0,0 +1,52 @@
+#
+# Hostname
+#
+
+
+# If there is an ssh connections, current machine name.
+function __sf_section_host -d "Display the current hostname if connected over SSH"
+
+  # ------------------------------------------------------------------------------
+  # Configuration
+  # ------------------------------------------------------------------------------
+
+  # --------------------------------------------------------------------------
+  # | SPACEFISH_HOST_SHOW | show hostname on local | show hostname on remote |
+  # |---------------------+------------------------+-------------------------|
+  # | false               | false                  | false                   |
+  # | always              | true                   | true                    |
+  # | true                | true                   | false                   |
+  # --------------------------------------------------------------------------
+
+  __sf_util_set_default SPACEFISH_HOST_SHOW true
+  __sf_util_set_default SPACEFISH_HOST_PREFIX "at "
+  __sf_util_set_default SPACEFISH_HOST_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+  __sf_util_set_default SPACEFISH_HOST_COLOR "blue"
+  __sf_util_set_default SPACEFISH_HOST_COLOR_SSH "green"
+
+  # ------------------------------------------------------------------------------
+  # Section
+  # ------------------------------------------------------------------------------
+
+  [ "$SPACEFISH_HOST_SHOW" = false ]; and return
+
+  if test $SPACEFISH_HOST_SHOW = 'always'; or set -q SSH_CONNECTION;
+    set host_color host
```

Comment:
In fish, variables are implicitly global unless the `-l` flag is used. 
> -l or --local forces the specified shell variable to be given a scope that is local to the current block, even if a variable with the given name exists and is non-local

As such I would suggest adding it not to clutter the namespace. ✌️

Reply:
I don't believe `host` has a value here.
I assume you might be trying to instantiate both variables, but in this case you're assigning the string "host" to `host_color`. 😉